Zilon.Core/Zilon.Bot.Players/BotActorTaskSourceBase.cs

```diff
@@ -74,17 +80,27 @@ public IActorTask[] GetActorTasks(IActor actor)
 
                 if (actorTask == null)
                 {
-                    return System.Array.Empty<IActorTask>();
+                    return Task.FromResult<IActorTask>(new IdleTask(actor, 1));
                 }
 
-                return new[] { actorTask };
+                return Task.FromResult(actorTask);
             }
             else
             {
                 _actorStrategies.Remove(actor);
             }
 
-            return System.Array.Empty<IActorTask>();
+            throw new System.Exception();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['System.Exception' should not be thrown by user code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5544638)